pkgs/development/ocaml-modules/data-encoding/default.nix

```
@@ -14,15 +14,18 @@
 , ppx_expect
 }:
 
-buildDunePackage {
+buildDunePackage rec {
   pname = "data-encoding";
-  version = "0.5.3";
+  version = "0.6";
```

```suggestion
  version = "0.7.1";
```

Upstream has released:
* v1.0.0: https://gitlab.com/nomadic-labs/data-encoding/-/tags/v1.0.0
* v0.7.1: https://gitlab.com/nomadic-labs/data-encoding/-/tags/v0.7.1
Any reason to be upgrading to `0.6` and not newer version?

---

The `ligo` package is not yet compatible with anything more recent than 0.6.